config.json

```diff
@@ -34,7 +34,7 @@
       "description": "MsBuild building options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/nologo /maxcpucount /verbosity:minimal"
+      "defaultValue": "/nologo /maxcpucount /verbosity:minimal /nodeReuse:false"
```

Comment:
I see, I thought this was like a default that might be consumed by corefx. In that case, then this LGTM.